toolkit/tools/pkg/imagecustomizerlib/customizepartitions.go

```diff
@@ -11,11 +11,11 @@ import (
 
 func customizePartitions(buildDir string, baseConfigPath string, config *imagecustomizerapi.Config,
 	buildImageFile string,
-) (string, error) {
-	if config.Disks == nil && config.SystemConfig.BootType == imagecustomizerapi.BootTypeUnset {
+) (bool, string, error) {
+	if !hasPartitionCustomizations(config) {
```

Comment:
No strong opinions, but curious why not a member function of the `Config` struct? I've gone both ways, mostly just curious.

Reply:
I generally like to keep my API types as free of code as possible.